ui/src/pages/backoffice/Document/DocumentRequestDetails/components/DocumentRequestMetadata/DocumentRequestMetadata.js

```diff
@@ -97,7 +97,14 @@ export default class DocumentRequestMetadata extends Component {
           </Grid.Row>
         </Grid>
         <Divider />
-        <RequestActions />
+        <RequestActions
+          requestPid={request.metadata.pid}
+          requestState={request.metadata.state}
+          renderRequestCancelHeader={pid => `Reject Document Request #${pid}`}
```

Comment:
This will be more complex https://github.com/inveniosoftware/invenio-app-ils/issues/634
Shall we keep it like this or implement later?

Reply:
I refactored this as, at least now, I used it in 2 different places...I think is fine to centralize it and make whatever changes needed in one place and reuse it...WDYT?